xep-0369.xml

```diff
@@ -580,28 +588,31 @@
     </section3>
 
     <section3 topic="Participants Node" anchor="participants-node">
-      <p>Each channel participant is represented as an item of the 'urn:xmpp:mix:nodes:participants' channel node.  Each item is named by the bare proxy JID of the participant. For example '123456#coven@mix.shakespeare.example' might name the node item associated with participant 'hag66@shakespeare.example'.  Information is stored in a &lt;participant/&gt; element qualified by the 'urn:xmpp:mix:core:0' namespace.   The nick associated with the user is optional and is stored in a &lt;nick/&gt; child element of the &lt;participant/&gt; element.   The nick for each channel participant MUST be different to the nick of other participants.
+      <p>Each channel participant is represented as an item of the 'urn:xmpp:mix:nodes:participants' channel node.  Each item is named by the Stable Participant ID of the participant. For example '123456' might name the node item associated with participant 'hag66@shakespeare.example'.  Information is stored in a &lt;participant/&gt; element qualified by the 'urn:xmpp:mix:core:0' namespace.   The nick associated with the user is optional and is stored in a &lt;nick/&gt; child element of the &lt;participant/&gt; element.   The nick for each channel participant MUST be different to the nick of other participants.
       </p>
       <p>
-       A Nick MAY be associated with a participant.  The nick associated with the user is stored in a &lt;nick/&gt; child element of the &lt;participant/&gt; element.   The nick for each channel participant MUST be different to the nick of other participants.
+       A Nick MAY be associated with a participant, which provides a user-oriented  description of the participant.  The nick associated with the user is stored in a &lt;nick/&gt; child element of the &lt;participant/&gt; element.   The nick for each channel participant MUST be different to the nick of other participants.
+      </p>
+      <p>
+        Where a nick is provided for a user, it is generally recommended to use this nick or the JID to display the user.   This enables consistent representation of participants for all participants in the channel.
```

Comment:
Surely we should only recommend showing the nick? JIDs should be hidden from users whenever possible, and in this case a client showing JIDs might have unpleasant interactions in anonymous rooms.

Reply:
&lt;Steve&gt; The JID will be known to all. I can see some situations where you’d prefer to use the definitive name. The key point is to not use local references. I’m fine to delete “or the JID” if you prefer.

Reply:
I think that would be better.